docs/ipr/scripting/RNA_Expression_Metrics.ipynb

```diff
@@ -0,0 +1,4089 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "RNA Expression Metrics.ipynb",
+      "provenance": [],
+      "collapsed_sections": [],
+      "toc_visible": true,
+      "authorship_tag": "ABX9TyPv5gZCKQiLgiRHh9j7xmLa",
+      "include_colab_link": true
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "view-in-github",
+        "colab_type": "text"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/bcgsc/pori/blob/feature%2Fcolab-notebooks/docs/ipr/scripting/RNA_Expression_Metrics.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "FgmSOXFHOgml"
+      },
+      "source": [
+        "# Calculating Expression Outliers Against a Reference Data Set\n",
+        "\n",
+        "## Download the Data\n",
+        "\n",
+        "First we need to download the dataset we will use. We are going to use the TCGA data here since it is publically available and well known ([UCSC Xena TCGA PANCAN](https://xenabrowser.net/datapages/?cohort=TCGA%20Pan-Cancer%20(PANCAN))). We will download the TPM matrix as well as the corresponding metadata file"
```

Comment:
maybe 'the TCGA' -> 'TCGA' here

Reply:
(also might be worth mentioning the tpms are actually log2(tpm + 0.001) - maybe that's obvious though)

Reply:
That is documented in the link we give so I think we can leave it out here.